.github/workflows/test.yml

```diff
@@ -23,7 +23,14 @@ jobs:
         with:
           node-version: '16'
       - run: npm ci
-      - run: npm test
+      - run: npm test-ci
```

Comment:
couldnt we just add -- --code-coverage instead of adding more scripts to npm?

Reply:
We could indeed.

Reply:
Would prefer that as too many scripts in package.json gets confusing fast